discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -71,7 +65,7 @@ def iso_download_path(self):
         return self._config.iso_download_path
 
     @property
-    def download_image(self):
+    def download_image_value(self):
```

Comment:
I'm afraid this property name is not indicative enough

Reply:
Me too.. but `download_logs` is worse.
I'm open for suggestions..

Reply:
what does it do?

Reply:
We need something with more boolean name

Reply:
Maybe ``enable_image_download``?